[2025-03-18T15:18:05.248+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-18T15:18:05.259+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2025-03-17T13:30:00+00:00 [queued]>
[2025-03-18T15:18:05.263+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2025-03-17T13:30:00+00:00 [queued]>
[2025-03-18T15:18:05.264+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-03-18T15:18:05.268+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): concat_data> on 2025-03-17 13:30:00+00:00
[2025-03-18T15:18:05.274+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=127) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-18T15:18:05.275+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'concat_data', 'scheduled__2025-03-17T13:30:00+00:00', '--job-id', '1692', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpahl7scln']
[2025-03-18T15:18:05.276+0000] {standard_task_runner.py:91} INFO - Job 1692: Subtask concat_data
[2025-03-18T15:18:05.276+0000] {standard_task_runner.py:64} INFO - Started process 140 to run task
[2025-03-18T15:18:05.295+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.concat_data scheduled__2025-03-17T13:30:00+00:00 [running]> on host b7aba691e1ff
[2025-03-18T15:18:05.328+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='concat_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-17T13:30:00+00:00'
[2025-03-18T15:18:05.329+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-18T15:18:05.331+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2025-03-18T15:18:05.355+0000] {logging_mixin.py:188} INFO -                OPEN     HIGH      LOW    CLOSE      VOLUME
DATE                                                      
2022-12-30  15436.0  15577.0  15296.0  15366.0   1339500.0
2023-01-03  15436.0  15927.0  15401.0  15927.0   1547700.0
2023-01-04  16033.0  16033.0  15822.0  15892.0   7244400.0
2023-01-05  15962.0  16138.0  15892.0  16033.0   2104500.0
2023-01-06  15998.0  16383.0  15857.0  16243.0   8403100.0
...             ...      ...      ...      ...         ...
2025-03-06  26250.0  26400.0  26150.0  26400.0   5991000.0
2025-03-07  26500.0  26800.0  26350.0  26650.0  12984900.0
2025-03-12  26500.0  26700.0  26200.0  26300.0   8814301.0
2025-03-14  26200.0  26200.0  25900.0  26000.0  12170100.0
2025-03-16  26200.0  26200.0  25900.0  26000.0  12170100.0

[524 rows x 5 columns]
[2025-03-18T15:18:05.463+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-18T15:18:05.463+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-18T15:18:05.466+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=concat_data, run_id=scheduled__2025-03-17T13:30:00+00:00, execution_date=20250317T133000, start_date=20250318T151805, end_date=20250318T151805
[2025-03-18T15:18:05.498+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-18T15:18:05.510+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-18T15:18:05.510+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
